dts/bindings/display/solomon,ssd1351.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+# SPDX-License-Identifier: Apache-2.0
+
+description: SSD1351 display controller
+
+compatible: "solomon,ssd1351"
+
+include: [mipi-dbi-spi-device.yaml, display-controller.yaml]
+
+properties:
+  display-offset:
+    type: int
+    required: true
+    description: Vertical offset by com from 0 ~ 127. Typically 128 - height.
+
+  start-line:
+    type: int
+    required: true
+    description: Start line of display RAM to be displayed by selecting a value from 0 to 127.
+      Typically 0.
+
+  multiplex-ratio:
+    type: int
+    required: true
+    description: Multiplex ratio from 0MUX to 127MUX. Typically same value as height - 1.
+
+  phase-length:
+    type: int
+    required: true
+    description: Phase 2 Period (7:4) and Phase 1 Period (3:0).
+
+  oscillator-freq:
+    type: int
+    required: true
+    description: Front clock divider (3:0) / oscillator frequency (7:4).
+      If you get weird scanlines, increase oscillator frequency and play with phase length values,
+      for example set this to 0xF1. Note this increases power consumption.
+
+  precharge-time:
+    type: int
+    required: true
+    description: Set 2nd precharge time for segments. A[3:0]. Typically 0x8.
```

Comment:
maybe elaborate on valid values (1 to 15 DCLKS, with 0 being invalid)?

<img width="283" alt="image" src="https://github.com/user-attachments/assets/082a7326-6582-478e-95e7-c920a39d3d99" />

Reply:
Done